spec/paperclip/storage/eitheror_spec.rb

```diff
@@ -79,27 +81,71 @@
 
       expect(user.avatar.some_unknown_method).to eq 'some response'
     end
+
+    context 'and an alias is set' do
+      it 'uses the aliased method' do
+        either_storage = user.avatar.instance_variable_get(:@either)
+        either_storage.stub(:either_handler)
+
+        expect(either_storage).to receive(:either_handler).with(:params)
+
+        user.avatar.only_on_or(:params)
+      end
+
+      context 'and the alias is to a lambda' do
+        it 'calls the lambda with both storages and any extra arguments' do
+          user.avatar.either_lambda_alias(:param)
+
+          either_storage = user.avatar.instance_variable_get(:@either)
+          or_storage = avatar.instance_variable_get(:@or)
+
+          expect(User.instance_variable_get(:@either_lambda_called_with)).to eql [either_storage, or_storage, user.avatar, :param]
```

Comment:
This was the first idea I had on how to verify this. 
Suggestions are REALLY welcome.